apps/directory/src/filter-config/facetConfigurator.js

```diff
@@ -68,8 +68,9 @@ export function createFilters(filters) {
       matchTypeForFilter:
         "any" /** if it has been selected from bookmark, it will be applied here. */,
       showMatchTypeSelector:
-        facet.showMatchTypeSelector ||
-        true /** if you want to make match all / match any available */,
+        facet.showMatchTypeSelector === undefined
+          ? true
+          : facet.showMatchTypeSelector /** if you want to make match all / match any available */,
```

Comment:
you could also use ` facet.showMatchTypeSelector ??  true`  This will assign only if the var has no value (so undefined or null)